modules/simple-payments/simple-payments.php

```diff
@@ -168,17 +168,8 @@ function parse_shortcode( $attrs, $content = false ) {
 
 		$data['id'] = $attrs['id'];
 
-		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-			require_once WP_CONTENT_DIR . '/lib/display-context.php';
-			$context = \A8C\Display_Context\get_current_context();
-			if ( \A8C\Display_Context\EMAIL === $context ) {
-				// Avoid enqueueing unsupported files by emails.
-				return $this->output_shortcode( $data );
-			}
-		}
-
 		if ( ! $this->is_enabled_jetpack_simple_payments() ) {
-			if ( ! is_feed() ) {
+			if ( jetpack_is_frontend() ) {
```

Comment:
This is rather unrelated to changes of this PR, but noted that we no longer need to use the display context lib here and decided to simplify the code.